recipe/meta.yaml

```diff
@@ -1,16 +1,13 @@
 {% set name = "yarn" %}
-{% set version = "1.13.0" %}
+{% set version = "1.15.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://registry.npmjs.org/{{ name }}/-/{{ name }}-{{ version }}.tgz
-  # this is published on npm
-  sha1: affa9c956739548024068532a902cf18c9cb523f
-  # this is calculated
-  sha256: 83277bd505c7f4009c13077266020c97298727de7edf67af5ca66eccae9d4632
+  sha1: 7a064ca81ca34235f16376ad2f796ed432f9e285
```

Comment:
Since this didn't fail any of the builds, it seems like specifying only the hash that the bot updates will make this easier to maintain.